frontend/package.json

```diff
@@ -65,7 +65,8 @@
     "redux-immutable": "^4.0.0",
     "redux-logger": "^3.0.6",
     "redux-saga": "^1.1.3",
-    "styled-components": "^5.3.5"
+    "styled-components": "^5.3.5",
```

Comment:
pls remove styled-components as ui framework; we only use materiual ui (mui5) for styling